config/routes.rb

```diff
@@ -1,11 +1,12 @@
-Rails.application.routes.draw do
-root 'users#index'
-  resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :new, :create, :show]
-  end
-
-  resources :posts do
-    resources :comments, only: [:create]
-    resources :likes, only: [:create]
-  end
-end
+Rails.application.routes.draw do
+  devise_for :users
+root 'users#index'

```

Comment:
- [ ] Your homepage/ root is **not properly routed** and therefore the homepage is showing a **blank screen**. The assigned root happens to have changed as per the _devise_ authentication setup. Kindly ensure to improve upon this aspect by using routing your home to
 `http://localhost:3000/users/sign_in`

Reply:
Thank you for the review, @rloterh.
This will be worked on immediately.
Cheers 👍

Reply:
Hi @rloterh,

Thank you for the review. 😀 
The change would be effected as soon as possible. However, kindly note that, according to CR standards, this was not just introduced in the new submission and could have been marked as **OPTIONAL** because it was missed by the previous CR.

I'll send in another review request as soon as possible.

Cheers 👍

Reply:
Hi @DammyShittu 
Kindly note that I'm requesting you to apply this fix because **it was impossible for me/ or other reviewers to see what changes you/ your team have implemented** with respect to the required changes by the previous code reviewer as your main page was broken.

Cheers, happy coding!